jenkins/jobs/image-apertis.yaml

```diff
@@ -0,0 +1,79 @@
+- job:
+    name: "image-apertis"
+    concurrent: false
+    description: Apertis images for LXC and LXD.
+    node: master
+    project-type: matrix
+
+    axes:
+    - axis:
+        name: architecture
+        type: slave
+        values:
+        - amd64
+        - arm64
+        - armhf
+
+    - axis:
+        name: release
+        type: user-defined
+        values:
+        - '17.12'
+        - '18.03'
+        - '18.06'
+        - '18.09'
+        - '18.12'
+        - v2019pre
```

Comment:
That's a pre-release snapshot so I don't think this should be included.

Reply:
It's been removed.